cocos/scripting/js-bindings/jswrapper/v8/Object.cpp

```diff
@@ -214,6 +214,17 @@ namespace se {
     Object* Object::createArrayBufferObject(void* data, size_t byteLength)
     {
         v8::Local<v8::ArrayBuffer> jsobj = v8::ArrayBuffer::New(__isolate, byteLength);
+        
+#if CC_TARGET_PLATFORM == CC_PLATFORM_OPENHARMONY
+        if (data)
+        {
+            memcpy(jsobj->GetBackingStore()->Data(), data, byteLength);
+        }
+        else
+        {
+            memset(jsobj->GetBackingStore()->Data(), 0, byteLength);
```

Comment:
So does openharmony depend on a v8 version which is different from other platforms?
Otherwise, why do we need to have a branch here?

Reply:
yes. openharmony use 9.1.269 version, other platforms use 8.0.426 version